functions/Copy-DbaCredential.ps1

```diff
@@ -154,7 +154,9 @@ function Copy-DbaCredential {
                     if (!$force) {
                         $copyCredentialStatus.Status = "Skipping"
                         $copyCredentialStatus.Notes = "Already exists on destination"
-                        $copyCredentialStatus | Select-DefaultView -Property DateTime, SourceServer, DestinationServer, Name, Type, Status, Notes -TypeName MigrationObject
+                        if ($Pscmdlet.ShouldProcess($destServer.Name, "Skipping $identity, already exists")) {
```

Comment:
Is `$identity`correct? Maybe `$credentialName` is better / correct. Same change in line 164 (new)

Reply:
And line 177 (new) - but line 178 (new) uses the value - so maybe `$identity` is correct?

Reply:
You're right! `$credentialName` was appropriate. But VS Code is acting so poorly, I'll have to fix it on the web 😡 

![image](https://user-images.githubusercontent.com/8278033/123264728-9a441e00-d4fa-11eb-8ccc-fc2676b84036.png)